testdata/src/go.uber.org/maps/lib/const.go

```diff
@@ -0,0 +1,3 @@
+package lib
+
+const MyStrConst = "xyz"
```

Comment:
Can we have a global variable here and corresponding test cases as well (such that it should not be considered stable)? Or is it already covered somewhere else (even so, I feel we can just add a simple case for sake of completeness)?

Reply:
That's fair. I have added a test case here.